rholang/src/main/scala/coop/rchain/rholang/interpreter/Runtime.scala

```diff
@@ -197,8 +199,100 @@ object Runtime {
         )
     }.sequence
 
+  object SystemProcess {
+    case class Context[F[_]: Async](
```

Comment:
why is Async needed? Why is any bound needed at all in here? I think it'll compile without that, giving us more flexibility and less assumptions to fulfill

Reply:
I guess I see - it's for the `SystemProcesses` to be created. I've just checked the constraint it requires can be lowered to just `Sync` by just changing it :). Can you do that?

Reply:
done